src/style/card__describe.css

```diff
@@ -0,0 +1,48 @@
+.card {
+  display: flex;
+  /* align-items: center; */
+}
+
+.card__title {
+  width: 166px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color_card-title);
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-style: normal;
+  font-weight: 400;
```

Comment:
font-style: normal;
  font-weight: 400;
is default values, so remove it everywhere